src/js/blight/BlightContainer.jsx

```diff
@@ -1,22 +1,42 @@
 import React, { Component } from "react";
+import _ from "lodash";
+
+import blightCards from "./blight-cards.json";
 
 export default class BlightContainer extends Component {
   constructor(props) {
     super(props);
     this.totalBlight = this.props.players * 2;
 
-    this.state = { numBlight: this.totalBlight };
+    this.state = { 
+      numBlight: this.totalBlight,
+      blightCard: {},
+      endGameTrigger: false
+    };
 
     this.handleAddBlight = this.handleAddBlight.bind(this);
     this.handleRemoveBlight = this.handleRemoveBlight.bind(this);
 
   }
+
   handleAddBlight(e) {
     this.setState({ numBlight: this.state.numBlight + 1 });
   }
   handleRemoveBlight(e) {
-    this.setState({ numBlight: this.state.numBlight - 1 });
+    const newBlight = this.state.numBlight - 1;
+    this.setState({ numBlight: newBlight});
+    if(newBlight <= 0 && this.state.endGameTrigger){
```

Comment:
Always a good practice to explicitly check values instead of relying on javascript's falsey logic (pun-intended). 
`this.state.endGameTrigger === true`